src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * this function changes screenshot in the options menu as per user's OS theme preference. By default the
+ * user would see a light-themed screenshot but if the user has selected a dark themed OS, the screenshot
+ * would also appear dark.
+ * 
+ * @private
+ * does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
+ * @param  {Object} darkQuery
+ */
+const dark = window.matchMedia("(prefers-color-scheme: dark)");
```

Comment:
Let's please move this line into the `init` method, so all code is in some function.